addon/types/index.d.ts

```diff
@@ -0,0 +1,2 @@
+import 'ember-source/types';
```

Comment:
in the v2-addon blueprint, we've been using a folder with more words 😅 
https://github.com/embroider-build/addon-blueprint/tree/main/files/__addonLocation__/unpublished-development-types

Reply:
Fixed

Reply:
we should update addon to sync with latest updates in addon-blueprint, I hope I can do it sooner than later.